.eslintrc.yaml

```diff
@@ -19,19 +19,10 @@ parserOptions:
   ecmaVersion: 2020
   sourceType: module
 rules:
-  indent:
-    - error
-    - 2
-  linebreak-style:
-    - error
-    - unix
```

Comment:
These rules were conflicting with Prettier! Fine most of the time, was causing some issues under my new code comments on `applyViteHtmlTransform`. Ran the eslint CLI tool [as detailed here](https://github.com/prettier/eslint-config-prettier#cli-helper-tool) to find what was safe to delete